autogen/retrieve_utils.py

```diff
@@ -200,26 +237,18 @@ def create_vector_db_from_dir(
             collection_name,
             get_or_create=get_or_create,
             embedding_function=embedding_function,
-            # https://github.com/nmslib/hnswlib#supported-distances
```

Comment:
Could we keep these info to hnswlib, maybe in the docstring if you prefer not have them in the comments?

Reply:
My bad, didn't mean to remove this. They have been re-added.